src/components/Gesture.svelte

```diff
@@ -57,35 +57,48 @@
 
   const gesturePlaceholderName = $t('content.data.classPlaceholderNewClass');
   const recordingDuration = get(settings).duration;
-  const countdownInitialValue = 3;
+  interface CountdownConfig {
+    value: string | number;
+    duration: number;
+    class?: string;
+  }
+  const countdownConfigs: CountdownConfig[] = [
+    { value: 3, duration: 500, class: 'text-8xl' },
+    { value: 2, duration: 500, class: 'text-8xl' },
+    { value: 1, duration: 500, class: 'text-8xl' },
+    { value: $t('content.data.recordingDialog.go'), duration: 1000, class: 'text-6xl' },
```

Comment:
Now the “Go” looks the same size as “Recording”, though it is actually slightly bigger (text-6xl instead of text-5xl). If they were the same size, “Go” strangely looks smaller (possibly due to the length of the word)